Launcher > Developing command `node index.js` fails

### Current behavior

[packages/launcher/README > Developing](https://github.com/cypress-io/cypress/blob/develop/packages/launcher/README.md#developing) instructions fail with
`code: 'MODULE_NOT_FOUND'`

The instructions are:

> To see browsers detected on your machine, just run:
>
>```bash
> node index.js
> ```

The file `packages/launcher/index.js` does not exist.

### Desired behavior

[packages/launcher/README > Developing](https://github.com/cypress-io/cypress/blob/develop/packages/launcher/README.md#developing) should give working instructions for contributors.

### Test code to reproduce

```shell
git clone https://github.com/cypress-io/cypress
cd cypress
yarn workspace @packages/launcher install
cd packages/launcher
node index.js
```


### Cypress Version

First reported on version: `13.11.0`
Still not fixed in: `15.6.0`

### Node version

~~`v18.17.1`~~ ~~`v20.18.1`~~ `v22.19.0` 

### Operating System

Ubuntu `22.04.4` LTS and `24.04.3` LTS

### Debug Logs

```text
$ node index.js
node:internal/modules/cjs/loader:1080
  throw err;
  ^

Error: Cannot find module '/home/mike/github/cypress-io/cypress/packages/launcher/index.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
    at Module._load (node:internal/modules/cjs/loader:922:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v18.17.1
```

### Other

- The instructions worked until https://github.com/cypress-io/cypress/pull/15646 was merged in Apr 2021 for Cypress `7.0.1`
- Attempting to use `node index.ts` instead leads to other errors.

--- a/packages/launcher/README.md
+++ b/packages/launcher/README.md
@@ -4,18 +4,18 @@ This package finds and launches browsers for each operating system.
 
 ## Developing
 
-The TypeScript source files are in [`lib`](/lib) folder.
+The TypeScript source files are in the [`lib`](/lib) folder.
 
-To see browsers detected on your machine, just run:
+To see the browsers detected on your machine, run the following from the `packages/launcher` directory. The `@packages/ts` require hook transpiles the TypeScript on the fly:
 
 ```bash
-node index.js
+node -r @packages/ts/register -e "require('./lib/detect').detect().then(console.log, console.error)"
 ```
 
-You can supply a list of binaries to test if they're browsers or not. Try running:
+You can also check whether a specific binary is recognized as a browser by passing its path:
 
 ```bash
-node index.js /bin/bash /usr/bin/chromium-browser
+node -r @packages/ts/register -e "require('./lib/detect').detectByPath(process.argv[1]).then(console.log, console.error)" /usr/bin/chromium-browser
 ```
 
 ## Testing

